src/mrpro/algorithms/reconstruction/_direct_reconstruction.py

```diff
@@ -0,0 +1,122 @@
+from __future__ import annotations
+
+from typing import Self
+
+from mrpro.algorithms._prewhiten_kspace import prewhiten_kspace
+from mrpro.algorithms.reconstruction import Reconstruction
+from mrpro.data._CsmData import CsmData
+from mrpro.data._DcfData import DcfData
+from mrpro.data._IData import IData
+from mrpro.data._kdata._KData import KData
+from mrpro.data._KNoise import KNoise
+from mrpro.operators._FourierOp import FourierOp
+from mrpro.operators._LinearOperator import LinearOperator
+
+
+class DirectReconstruction(Reconstruction):
+    """Direct Reconstruction by Inverse Fourier Transform."""
+
+    def __init__(
+        self,
+        fourier_operator: LinearOperator,
+        csm: None | CsmData = None,
+        noise: None | KNoise = None,
+        dcf: DcfData | None = None,
+    ):
+        """Initialize DirectReconstruction.
+
+        Parameters
+        ----------
+        fourier_operator
+            Instance of the FourierOperator which adjoint is used for reconstruction.
+        csm
+            Sensitivity maps for coil combination. If None, no coil combination will be performed.
+        noise
+            Used for Prewhitening
+        dcf
+            Density compensation. If None, no dcf will be performed.
+            Also set to None, if the FourierOperator is already density compensated.
+        """
+        super().__init__()
+        self.FourierOperator = fourier_operator
+        # TODO: Make this buffers once DataBufferMixin is merged
+        self.dcf = dcf
+        self.csm = csm
+        self.noise = noise
+
+    @classmethod
+    def from_kdata(cls, kdata: KData, noise: KNoise | None = None, coil_combine: bool = True) -> Self:
```

Comment:
maybe "do_coil_combine" to highlight that this is a bool?

Reply:
but this is already given by the type?